index.html

```diff
@@ -163,16 +163,37 @@ <h2>About me</h2>
                 </div>
             </section>
 
-             <div>
+            <section class="form-section">
+             <form action="https://formspree.io/f/mjvzwlqy" method="post">
                 <h2 class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-                <br>
-                <div class="textareas">
-                <textarea name="name" id="" cols="41" rows="2">Full name</textarea>
-                <textarea name="e-mail" id="" cols="41" rows="2">E-mail address</textarea>
-                <textarea name="text" id="" cols="41" rows="10">Enter text here</textarea>
-                <div class="butt"><button class="btn" type="button"> Get in touch</button></div>
-                </div>
-            </div>
+                <ul>
+                    
+                    <li >
+                        <label for="name">
+                            <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                        </label>
+                    </li>
+                    <li>
+                        <label for="mail">
+                            <input type="e-mail" id="mail" name="user_email" placeholder="Email address" required>
+                        </label>
+
+                    </li>
+                    <li>
+                        <label for="msg">
+                            <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                        </label>
+                    </li>
+                    <li class="button">
+                        <button type="submit" class="btn">Get in touch</button>
+                    </li>
+
+
+                </ul>
```

Comment:
- [ ] Kindly make sure the **Get in touch** button follows as much as possible to the Figma design. You should also the different interactions such as `hover`, `active` (when it's pressed), `enable` (default state),... as shown in the Figma template on the image below
![Screenshot from 2022-09-27 14-10-20](https://user-images.githubusercontent.com/83042742/192522809-1bfc3ca5-d38f-4d1a-b57d-3f7b6320b61f.png)